gateway/settings.gradle

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Similar to above why are the ids for liquibase, eclipse, etc. "missing" Or is it fine?
![image](https://user-images.githubusercontent.com/65814168/133823631-a61c2148-8834-456b-b2c5-3ff3b144a5b9.png) [settings.gradle]

Reply:
Same for liquibase as for hibernate.
I am not sure what the eclipse plugin is used for. I can add it, it's not a problem. Are you familiar what it is used for?

Reply:
Unfortunately not really, I assume the eclipse plugin is used to maybe support Orion or similar things, so you can use Artemis directly in your IDE but I am not sure.